### ISSUE TITLE
Quoted value of a formula cell

### ISSUE THREAD

**Issue by antone-sb** [monitorjbl/excel-streaming-reader#64]
Hi! I've noticed that for a formula cell the value is returned in quotes. Is this by design? What is the reason for it? In particular, if the value is an empty string, two quote marks are returned.
[SampleSum.xlsx](https://github.com/monitorjbl/excel-streaming-reader/files/627046/SampleSum.xlsx)
 I get the following values with attached file:
-4 "minus"
4   ""

**Comment by apixandru** [comment]
Good catch, I have noticed similar behavior with the numbers, it should be fixed with this pull request https://github.com/monitorjbl/excel-streaming-reader/pull/118

Main file [Main.java.txt](https://github.com/monitorjbl/excel-streaming-reader/files/1421117/Main.java.txt) Input [SampleSum2.xlsx](https://github.com/monitorjbl/excel-streaming-reader/files/1421112/SampleSum2.xlsx)

Before #118 
```
-4.0      "minus"        "minus"        minus      
 4.0           ""             ""      "minus"      
-4.0      "minus"      ""minus""      "minus"      
```

After #118 
```
-4.0      minus        minus        minus      
 4.0                              "minus"      
-4.0      minus      "minus"      "minus"      
```

Native Apache Poi
```
-4.0      minus        minus        minus      
 4.0                              "minus"      
-4.0      minus      "minus"      "minus"      

```
